example-mapfragmentandview/SampleMapFragment/src/main/java/mil/emp3/examples/samplemapfragment/MainActivity.java

```diff
@@ -57,7 +54,7 @@ public void onEvent(MapStateChangeEvent mapStateChangeEvent) {
                         Log.d(TAG, "mapStateChangeEvent map2 " + mapStateChangeEvent.getNewState());
                         try {
                             onMapReady(map2);
-                            map2.setCamera(CameraUtility.buildCamera(33.9424368, -118.4081222, 2000000.0), false);
+                            map2.setCamera(CameraUtility.buildCamera(40.7128, -74.0059, 2000000.0), false);
```

Comment:
One map shows up on west coast and other on east coast. Just make sure user realizes that these are two different maps.